Source/Documentation/Manual/features-rollingstock.rst

```diff
@@ -318,6 +318,79 @@ for the purposes of calculation.
 - ``MU ( 3 )`` Locomotive must be in a different group to the lead locomotive
 
 
+Multiple type locomotive light glows
+------------------------------------
+
+Introduction
+''''''''''''
+
+As a default all OR (and MSTS) locomotives use the same texture to reproduce 
+the glow of their lights. This however doesn't allow to easily implement non-round 
+or LED array lights, neither to provide special glowing effects.
+
+This feature allows to specifiy customized light glow textures for the locomotives. 
+If nothing is specified, the standard light glow texture is used. Moreover in the ``Content`` 
+folder, two light glow textures are present: the "historical" one, and a new one, 
+more realistic. As a default the "historical" light glow texture is used, for backwards 
+compatibility; however adding a line to the Lights block in the .eng file the "new" light 
+glow texture is taken. Customized light glow textures can be either used for all lights 
+of a loco, or only for a subset of them. Different lights in the same locomotive 
+may have different customized light glow textures.
+
+Detailed spec
+'''''''''''''
+
+
+1) In the ``Content`` folder there is the default ``LightGlow.png``, which is displayed if 
+   no changes are done to the .eng file.
+2) In such folder there is also an ``ORTSLightGlow.png``, which is maybe more realistic.
+3) adding a line within the .eng file it is possible to select either ORTSLightGlow.png or any other .png. 
+   Here an example for the legacy Acela loco::
+
+    	Lights	( 17
+	        ORTSGraphic ( "ORTSLightGlow.png")
```

Comment:
Multiple of these ORTSGraphic lines were typed with slightly inconsistent formatting that's bugging me, please add a space between the " and ) to match the space present between ( and "

Reply:
OK, but it seems to me that OR uses the other typing, that is without parentheses. I'll modify the .rst file that way.